tests/test_copy.py

```diff
@@ -153,3 +153,11 @@ def test_pretend_option(dst):
     assert not (dst / "doc").exists()
     assert not (dst / "config.py").exists()
     assert not (dst / "pyproject.toml").exists()
+
+
+def test_subdirectory(dst):
+    render(dst, subdirectory="doc")
+    assert not (dst / "doc").exists()
+    assert not (dst / "config.py").exists()
+    assert (dst / "images").exists()
+    assert (dst / "manana.txt").exists()
```

Comment:
I want to remove the `dst` fixture at some point. Please don't use it for new code:

```suggestion
def test_subdirectory(tmp_path: Path):
    render(tmp_path, subdirectory="doc")
    assert not (tmp_path / "doc").exists()
    assert not (tmp_path / "config.py").exists()
    assert (tmp_path / "images").exists()
    assert (tmp_path / "manana.txt").exists()
```

Reply:
Woops, sure, sorry, missed that one